2.6/Dockerfile.rhel8

```diff
@@ -1,12 +1,12 @@
-FROM ubi8/s2i-base
+FROM ubi8/s2i-base:rhel8.1
```

Comment:
I have a feeling that this tag might only be available in our internal registry

Reply:
Hmm. Does that affect us in any way? Or is this supposed to work on CI without internal access? Isn't `ubi8/s2i-base` restricted also?

Reply:
Afaik the tag will not be available in public registries even after 8.1 GA (and neither is the s2i-base image built with this tag from upstream repositories), so while it does not affect us right now (as the image cant be built anyway) it will be the cause of failures later.

> Isn't ubi8/s2i-base restricted also?

What do you mean by that?

Reply:
Well, what I meant is that I didn't know whether `ubi8/s2i-base` is available at all. Simple test, I hope, proved it's accessible allright:
```console
$ podman pull ubi8/s2i-base

Trying to pull docker.io/ubi8/s2i-base...
  denied: requested access to the resource is denied
Trying to pull registry.fedoraproject.org/ubi8/s2i-base...
  manifest unknown: manifest unknown
Trying to pull quay.io/ubi8/s2i-base...
  unauthorized: access to the requested resource is not authorized
Trying to pull registry.access.redhat.com/ubi8/s2i-base...
Getting image source signatures
Copying blob 315fa5211806 done
Copying blob cb3c77f9bdd8 done
Copying blob 3579587f9454 done
Copying blob fd8daf2668d1 done
Copying config 385c47c21f done
Writing manifest to image destination
Storing signatures
385c47c21f71177dd82dc8e5bd2a8bf6478b642f686260d5932a4689f0c92d2a
```
And TMK I'm not authenticated. But then, I'm on internal network...

To avoid the possible failures would you rather have me use `:latest`?